Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageExBase.Placeholder.cs

```diff
@@ -52,6 +50,29 @@ public ImageSource PlaceholderSource
             set { SetValue(PlaceholderSourceProperty, value); }
         }
 
+        private static void PlaceholderSourceChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            var control = d as ImageExBase;
+
+            if (e.OldValue == null || e.NewValue == null || !e.OldValue.Equals(e.NewValue))
+            {
+                if (control is RoundImageEx round)
```

Comment:
@WilliamABradley If you want us to merge your branch, I recommend you to retarget your code to C# 6 only. To clarify, I am not against C# 7 but the fact is that the solution will not compile because CI does not support C# 7 for the moment.